application/src/main/kotlin/com/studentcenter/weave/application/meetingTeam/port/outbound/MeetingTeamInvitationRepository.kt

```diff
@@ -0,0 +1,12 @@
+package com.studentcenter.weave.application.meetingTeam.port.outbound
+
+import java.util.*
+
+interface MeetingTeamInvitationRepository {
+    fun save(
+        teamId: UUID,
+        invitationCode: UUID,
+        expirationSeconds: Long,
```

Comment:
p3) 소요시간을 표현할 때 Duration이 좋을 것 같아요
```suggestion
        expirationDuration: Duration,
```